### ISSUE TITLE
[QUESTION] Does wp-cron fire upon installation?

### ISSUE THREAD

**Issue by jayhill90** [lucatume/wp-browser#348]
I have an mu-plugin that has a cron setup that calls an API and writes data to a file. Since my API credentials are not yet configured in WordPress, I'm seeing that upon installation the code to create the file is triggered and this results in a blank file. My tests pass, but PHP warnings are thrown because the file's data is gone. Is it possible to disable cron's from firing at all during test execution?

**Comment by lucatume** [comment]
Hi @jayhill90,

I need more information to know what the issue is.

What modules is the suite using? WPLoader? Other?

Could you post the suite configuration file for me to have a better grasp of the issue? wp-cron should be deactivated during integration tests by the WPLoader module, if that is not the case, then that is a regression on an issue I had fixed previously.

**Comment by jayhill90** [comment]
For my wpunit tests we're running 

```
actor: WpunitTester
modules:
    enabled:
        - WPLoader
        - Asserts
        - \Helper\Wpunit
```

Basically when the install process fires off in Codeception to install WordPress, it seems the cron I have to create the file is triggered during installation, and so I end up with a blank file. I then get some undefined index errors until my _before() function to create the file again (basically mimicking the cron having actually been run once all my data is added as an option) and then my tests pass.

Basically I'm just trying to figure out why my file is being overwritten when installing WordPress.

**Comment by lucatume** [comment]
Would you be able to post the whole configuration file? A minimal, reproducible, example would help me.

**Comment by jayhill90** [comment]
I was able to get around it and resolve the issues I was having. Thanks though @lucatume!

**Comment by lucatume** [comment]
@jayhill90 that's great to hear.

If there is something I could add in the suite, or update to make the workaround less tricky or to make something better, let me know.
